db/db_options_test.cc

```diff
@@ -117,6 +117,87 @@ TEST_F(DBOptionsTest, GetLatestCFOptions) {
             GetMutableCFOptionsMap(dbfull()->GetOptions(handles_[1])));
 }
 
+TEST_F(DBOptionsTest, SetBytesPerSync) {
+  const int kValueSize = 1024 * 1024 * 8;
+  Options options;
+  options.create_if_missing = true;
+  options.bytes_per_sync = 1024 * 1024;
+  options.use_direct_reads = false;
+  options.write_buffer_size = 50 * kValueSize;
```

Comment:
Set write_buffer_size to a significant larger value (times 10?).  Same for the other test.

Reply:
Mind update write_buffer_size to a larger number for both of the tests? It will make the tests more deterministic.